src/server/products/get-productsSliderMain.server.ts

```diff
@@ -0,0 +1,37 @@
+'use server'
+import { connectToDb } from '@/server/connectToDb'
+import { Product } from '@/server/products/productSchema'
+import { IProductLocal, IVariableProduct } from '@/types/product'
+
+export const getProductsSliderMain = async (): Promise<IProductLocal[]> => {
+  console.time('fetch-products')
+
+  try {
+    await connectToDb()
+
+    const products = (
+      await Product.find()
+        .sort({ visited: -1 })
+        .limit(7)
+        .lean<IProductLocal[]>()
+    ).map((product) => ({
+      ...product,
+      name: product.name,
```

Comment:
what card is used as far as I know you get the correct array 
if you have sam error you can say it

Reply:
Fixed
or I misunderstood